src/main.cpp

```diff
@@ -2040,6 +2049,16 @@ void setup() {
         pinMode(PIN_STATUSLED, OUTPUT);
     }
 
+    #if WATER_SENS == 1
+        //NPN
+        pinMode(PIN_WATERSENSOR, INPUT_PULLUP);
+    #endif
+
```

Comment:
Can be shortened to #if ... #elif ... #endif
I know it's not consistent in the codebase but while we're at it :)